app/src/test/java/ch/epfl/qedit/model/MultiFieldFormatTest.java

```diff
@@ -0,0 +1,37 @@
+package ch.epfl.qedit.model;
+
+import static org.junit.Assert.assertNotEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import ch.epfl.qedit.model.answer.AnswerFormat;
+import ch.epfl.qedit.model.answer.MatrixFormat;
+import ch.epfl.qedit.model.answer.MultiFieldFormat;
+import org.junit.Test;
+
+public class MultiFieldFormatTest {
```

Comment:
Maybe the field to enter the number of questions should be explicit here. But that's just a note for future PRs, I think it is fine like this for this PR.